docs/administration.md

```diff
@@ -310,6 +311,10 @@ value set in `-zn` or `--zip-name`.
 If `--data-only` is provided, only the database will be exported. This option is intended
 to facilitate database upgrades without needing to clean documents and thumbnails from the media directory.
 
+If `--no-progress-bar` is provided, the progress bar will be hidenn, rendering the 
```

Comment:
```suggestion
If `--no-progress-bar` is provided, the progress bar will be hidden, rendering the 
```

Reply:
Ooof, of course I couldn't see the forest for the trees… Thanks for catching the typo!